Ed-Fi-Alliance-OSS/Ed-Fi-ODS-AdminApp
# [AA-1548] Admin API E2E Test Expansion

Update Admin API Postman Collection to include tests for `/applications` endpoints + expand `/vendors` tests to match new patterns and increase coverage. Besides test coverage, the main goal here is to allow future endpoints to follow the request pattern and copy scripts in a similar manner, to make writing them easier.

- adds requests / assertions per-response (OK, Bad Request, Not Found)
- asserts against response result values for accuracy
- includes fixes to API where tests failed
  - correct 404 response from `/applications/reset-credentials`
  - format of `/connect/register`

**Testing**

- make sure you're running in "Production" mode for error handling
  - added a new launch profile for VS
  - manually you can set `ASPNETCORE_ENVIRONMENT`, but other settings need to be updated as well
- follow the Readme:
  1. upload environment and collection
  2. update environment
  3. run collection
